src/table/settings/settings.ini

```diff
@@ -232,6 +232,19 @@ strhelp  = STR_CONFIG_SETTING_SUBSIDY_MULTIPLIER_HELPTEXT
 strval   = STR_SUBSIDY_X1_5
 
 [SDT_VAR]
+var      = difficulty.subsidy_duration
+type     = SLE_UINT16
+from     = SLV_CUSTOM_SUBSIDY_DURATION
+def      = 1
+min      = 0
+max      = 5000
+interval = 1
+str      = STR_CONFIG_SETTING_SUBSIDY_DURATION
+strhelp  = STR_CONFIG_SETTING_SUBSIDY_DURATION_HELPTEXT
+strval   = STR_JUST_INT
```

Comment:
I think this shouldn't be `STR_JUST_INT`, but more something that says: `1 year`. So it is more clear what the unit is?